sky/utils/timeline.py

```diff
@@ -18,6 +18,7 @@
 from sky.utils import locks
 
 _events = []
+_events_file_path = os.environ.get('SKYPILOT_TIMELINE_FILE_PATH')
```

Comment:
Not sure if it is relevant but caching the env var will make `timeline` no longer honor env override from client, so 

```
SKYPILOT_TIMELINE_FILE_PATH=test.json sky launch --cloud kubernetes -y
```

will no longer work